src/components/AccountForms/RegistrationForm/RegistrationForm.test.js

```diff
@@ -19,26 +19,53 @@ describe('<RegistrationForm />', () => {
     );
   });
 
+  afterEach(() => {
+    cleanup();
+  });
```

Comment:
Let's remove this, no longer needed with the latest version of RTL. Cleanup is done automatically for us behind the scenes.